onnx/defs/data_propagators.h

```diff
@@ -11,6 +11,8 @@ inline void appendDimToTensorShapeProto(TensorShapeProto& tsp, const TensorShape
     tsp.mutable_dim()->Add()->set_dim_value(dim.dim_value());
   } else if (dim.has_dim_param()) {
     tsp.mutable_dim()->Add()->set_dim_param(dim.dim_param());
+  } else {
+    tsp.mutable_dim()->Add();
```

Comment:
Thanks for fixing this! This looks correct. But I think the whole function can be simplified to
```
   * tsp.add_dim() = dim;
```

Reply:
Thank you!